.github/workflows/deploy-geo-prod.yml

```diff
@@ -0,0 +1,67 @@
+name: ⭐️ Deploy PLATEAU Geo production
+on:
+  workflow_dispatch:
+env:
+  IMAGE: ghcr.io/eukarya-inc/reearth-plateauview/plateauview-geo:latest
+  IMAGE_GCP: asia.gcr.io/reearth-plateau/plateauview-geo:latest
+  IMAGE_HUB: eukarya/plateauview3-geo:latest
```

Comment:
@rot1024 
Do you think this name is good?

Reply:
plateauview-geo is better i think